src/include/gurt/debug.h

```diff
@@ -302,6 +302,12 @@ do {									\
 #define DF_U64		"%" PRIu64
 #define DF_X64		"%" PRIx64
 
+#define D_DEPRECATED(replacement)					\
```

Comment:
(style)  Macros with multiple statements should be enclosed in a do - while loop
(style)  macros should not use a trailing semicolon